frequencyChar.js

```diff
@@ -0,0 +1,20 @@
+function itemFrequency(array) {
```

Comment:
By convention, the name of the file is the same name as the function or object being exported. So you'd want to name the file `itemFrequency.js`.